### PR TITLE
Adding manure module configs for open lots and harrowing

### PR DESCRIPTION
**Pull request by ltp23**
Issue #1129 

This PR does the following:

- Add a section in `manure_management.json` for the harrowing manure handler type. All the parameters are set to 0.
- Add a section in `manure_management.json` for the open lots treatment. I only kept the removal efficiency parameters and set most of them to 0 except the two for phosphorus and potassium. This is because I saw only these two removal efficiencies were used in the calculations. However, I used very small numbers for theses two (0.01) because I didn't know what values to use.
- Add descriptions in the metadata files for existing properties of manure treatment configs.

### CODE REVIEW

File: input/data/animal/default_animal.json [hunk 1]
```diff
@@ -179,7 +179,7 @@
       "housing_type": "open air barn",
       "pen_type": "tiestall",
       "max_stocking_density": 1.2,
-      "manure_management_scenario_id": 2
+      "manure_management_scenario_id": 8
```
Comment (ew3361zh) [inline input/data/animal/default_animal.json]: Are these meant to replace the standard manure_management_scenario_ids used?

**Comment by ew3361zh** [comment]
@ltp23 can you add what issue this is addressing to the description?

File: input/data/animal/default_animal.json (hunk 1 above)
Comment (tomhuhh) [reply input/data/animal/default_animal.json]: My understanding is that those are temporarily used for reviewing purposes.

**Review by tomhuhh — COMMENTED** [review]
LGTM. Thanks Loi! 

The code runs without difficulty for me, which means the added manure handling and storage methods work.  

The added descriptions in the metadata file seem to be fine. 

The only thing that I am uncertain about is the removal efficiency of phosphorus and potassium. I'm not sure if there is a design doc or  pseudocode doc that we can refer to. 

Thanks!

**Comment by tomhuhh** [comment]
I would like to add that for manure handling methods such as harrowing and tillage, there is currently no calculation code available. I wonder if this is intentional? Thanks! 

Screenshot 2024-01-30 at 4 54 07 PM

File: input/data/animal/default_animal.json (hunk 1 above)
Comment (ltp23) [reply input/data/animal/default_animal.json]: Right, they are there for reviewing purposes.

**Comment by ltp23** [comment]
> The only thing that I am uncertain about is the removal efficiency of phosphorus and potassium. I'm not sure if there is a design doc or pseudocode doc that we can refer to.

Thanks Haowen. I don't have the design doc to see what the reference values are. I put in the small values for those two just to signify that those two are used in the calculations.

**Comment by ltp23** [comment]
> I would like to add that for manure handling methods such as harrowing and tillage, there is currently no calculation code available. I wonder if this is intentional? Thanks!

Right, Haowen. All the manure handler classes inherit the base class and perform the same calculations, though the numbers can vary. These subclasses are empty currently because we don't have specific logic yet for each kind other than the calculations in the daily update function, which is done in the base class.

**Comment by KFosterReed** [comment]
> > The only thing that I am uncertain about is the removal efficiency of phosphorus and potassium. I'm not sure if there is a design doc or pseudocode doc that we can refer to.
> 
> Thanks Haowen. I don't have the design doc to see what the reference values are. I put in the small values for those two just to signify that those two are used in the calculations.

Yes, these removal efficiencies should all be 0

**Review by tomhuhh — APPROVED** [review]
Thanks all for the clarifications!

**Review by ew3361zh — APPROVED** [review]
LGTM!

**Pull request merged**